apps/util/lib/util.ex

```diff
@@ -92,6 +92,12 @@ defmodule Util do
     |> Date.to_string()
   end
 
+  def convert_using_timezone(time, time_zone) do
```

Comment:
Love it! This could use a `@spec` and a `@doc`, but thank you for pulling  this out into a function.